src/about/Intro.js

```diff
@@ -7,21 +7,76 @@ import urls from '../assets/urls'
 const Intro = (props) => {
   return (
     <div className="intro">
-      <h2>A team of blockchain engineers committed to scaling Ethereum.</h2>
-      <p>
-        We are implementing the first <a className="pink" href={urls.sharding}>sharding</a> client 
-        for the go-ethereum project.
-      </p>
-      <p>
-        Currently, blockchains do not scale. With Ethereum receiving over 
-        <span className="pink"> a million transactions per day</span>, the network is desperate for 
-        a solution that will help sustain its massive growth and adoption.
-      </p> 
-      <p>
-        We are building out the first sharding implementation for the
-        <a className="pink" href={urls.geth}> Geth (go-ethereum)</a> client, allowing the system to 
-        process transactions at over 10x - 100x of its current speed!
-      </p>
+      <h2>JOIN US IN OUR QUEST TO #SHARDL ETHEREUM!</h2>
+      <p>Welcome Friend!</p>
+      <p>There is an important matter at hand...</p>
+      <p>And there is little time, indeed.</p>
+      <p>You see, Ethereum's network has grown substantially in the past year, and it doesn't look to be stopping anytime soon. The community is growing stronger each day, and more dApps are being envisioned and deployed by the second!</p>
+      <p>But there's one little problem...</p>
+      <p>And that is this: Public blockchains, in their current state, simply do not scale!</p>
+      <p><span className="pink"><em>With Ethereum receiving over a million transactions per day (!)</em></span>, the network is desperate for a solution that will help sustain its massive growth and adoption.</p>
+      <p>We believe Ethereum will one day be the the home of amazing dApps that are relied on by billions of users.</p>
+      <p>A decentralized version of Uber, for instance.</p>
+      <p>Yet Ethereum's ENTIRE NETWORK capacity right now is ~15 transactions per second.</p>
+      <p>Visa processes 45,000 transactions per second.</p>
+      <p>Supporting a decentralized Uber, Visa, AirBnB and others simply is not feasible at this time on Ethereum...</p>
+      <p>But there is an answer!</p>
+      <p>It's called "sharding".</p>
+      <p>The concept of sharding is based on splitting the entire state of the network into a bunch of partitions called shards that contain their own independent piece of state and transaction history. In this system, certain nodes would process transactions only for certain shards, allowing the throughput of transactions processed in total across all shards to be much higher than having a single shard do all the work as the mainchain does now.</p>
+      <p>You can read more about sharding in <a className="blue" href={urls.shardingExplained}>this blog post we put together</a>.</p>
+      <p>But don't leave! Bookmark that link, and keep reading on...</p>
+      <p>You see, sharding is one of, if not THE, most important thing for Ethereum's future.</p>
+      <p>And that's where Prysmatic Labs comes in.</p>
```

Comment:
I think we can eliminate everything up to this point. That much background isn't necessary as this site isn't aimed towards the mainstream. I'm more of an advocate for our current tone and to  keep it consistent.